SimG4Core/Application/src/StackingAction.cc

```diff
@@ -260,7 +260,8 @@ G4ClassificationOfNewTrack StackingAction::ClassifyNewTrack(const G4Track * aTra
 	  ((pdg == 2112) && (ke < kmaxNeutron))) { classification = fKill; }
     }
     if (!trackNeutrino  && classification != fKill) {
-      if (pdg == 12 || pdg == 14 || pdg == 16 || pdg == 18) 
+      if (pdg == 12 || pdg == 14 || pdg == 16 || pdg == 18 ||
```

Comment:
@bsunanda , @davidlange6 , this code is called for each track once, so from point of view of CPU this is the second order effect - not at each step. However, it seems it can be a bit optimised, because in production we check partly some exotic conditions. Clean-up (reduction of "if" statements ) will likely bring a little but if you agree I can do this in the following PR.